packages/jellyfin-media-player-deb/jellyfin-media-player-deb.pacscript

```diff
@@ -0,0 +1,39 @@
+name="jellyfin-media-player-deb"
+gives="jellyfin-media-player"
+pkgver="1.9.1"
+arch=('amd64')
+breaks=("${name}-bin" "${name}" "${name}-app")
+pkgdesc="Jellyfin desktop client using jellyfin-web with embedded MPV player"
+codename="$(lsb_release -cs)"
```

Comment:
Use Pacstall's builtin `${DISTRO}` variable, it looks like this `ubuntu:jammy`
```suggestion
codename="${DISTRO#*:}"
```

Reply:
Thank you very much for the advice, I didn't know about the existence of the builtin variable `${DISTRO#*:}`, I had seen `$(lsb_release -cs)` from other pacscripts.